<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mRender</class>
 <widget class="QMainWindow" name="mRender">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mRender</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QOpenGLWidget" name="renderWidget"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="renderButton">
         <property name="text">
          <string>Render</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>230</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>230</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_1">
          <attribute name="title">
           <string>Shape</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>-1</x>
             <y>-1</y>
             <width>221</width>
             <height>221</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QRadioButton" name="squareRadioButton">
              <property name="text">
               <string>Square</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rectangleRadioButton">
              <property name="text">
               <string>Rectangle</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="pentagonRadioButton">
              <property name="text">
               <string>Pentagon</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="triangleRadioButton">
              <property name="text">
               <string>Triangle</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="trapRadioButton">
              <property name="text">
               <string>Trapezoid</string>
              </property>
             </widget>
            </item>
           </layout>
           <zorder>pentagonRadioButton</zorder>
           <zorder>squareRadioButton</zorder>
           <zorder>rectangleRadioButton</zorder>
           <zorder>triangleRadioButton</zorder>
           <zorder>trapRadioButton</zorder>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Color</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>221</width>
             <height>221</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QRadioButton" name="redRadioButton">
              <property name="text">
               <string>Red</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="yellowRadioButton">
              <property name="text">
               <string>Yellow</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="greenRadioButton">
              <property name="text">
               <string>Green</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="blueRadioButton">
              <property name="text">
               <string>Blue</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="whiteRadioButton">
              <property name="text">
               <string>White</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>renderButton</sender>
   <signal>clicked()</signal>
   <receiver>mRender</receiver>
   <slot>render()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>squareRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>mRender</receiver>
   <slot>updateShape()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rectangleRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>mRender</receiver>
   <slot>updateShape()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pentagonRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>mRender</receiver>
   <slot>updateShape()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>triangleRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>mRender</receiver>
   <slot>updateShape()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trapRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>mRender</receiver>
   <slot>updateShape()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>mRender</receiver>
   <slot>updateColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yellowRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>mRender</receiver>
   <slot>updateColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>mRender</receiver>
   <slot>updateColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>mRender</receiver>
   <slot>updateColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>whiteRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>mRender</receiver>
   <slot>updateColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>render()</slot>
  <slot>updateShape()</slot>
  <slot>updateColor()</slot>
 </slots>
</ui>
